pkg/scaffold/templates.go

```diff
@@ -351,15 +314,15 @@ var (
 // for display to an end user.
 func ValidateProjectName(s string) error {
 	if s == "" {
-		return errors.New("A project name may not be empty")
+		return errors.New("a project name may not be empty")
```

Comment:
not "may", but "must".
delete "a" is better, or "a project's name", the same for the below.